o4/o4.py

```diff
@@ -21,16 +21,17 @@
                 provide -f. For a faster check from 80% of current changelist,
                 add -q (recommended).
   clean         Clean <path>.
-  <path>        Specify perforce style path, optionally specify "@changelist", if not given, head
-                will be determined. If path is a directory, "/..." is implied.
+  <path>        Specify perforce style path, optionally specify "@changelist";
+                if not given, head will be determined. If path is a directory,
+                "/..." is implied.
                 This path must always be a directory, not a file.
```

Comment:
```suggestion
                if not given, head will be determined. This path must always be 
                a directory, not a file.  Since path is a directory, "/..." is implied.
```